docs/Installation.md

```diff
@@ -73,3 +73,19 @@ Test if installation is succesful by compiling a test
 cd test
 make TEST=blinky
 ```
+Now that you've successfully compiled a test, we want to flash our code to the microcontroller. The term "flashing" in an embedded system refers to putting your code onto the microcontroller. If you're using WSL there are some extra setup steps you need to do to connect to the microcontroller, that can be found [here](./FlashAndTheBug.md) in the `Attaching USB devices in WSL` section.  
+
+After that run ``lsusb`` and make sure you see an ST-Link Debug device is seen by your terminal. Then run the following command to flash to your microcontroller
+```sh
+make flash
+```
+
+# Common Errors
+
+## Could not open USB device
```

Comment:
Not sure if this has been resolved but conflicting bin files means you have to `make clean` everytime you flash. That's a common error imo.